### ISSUE TITLE
Test parameter save/load related to the BI Revive System.

### ISSUE THREAD

**Issue by Wyqer** [Wyqer/kp_liberation#364]
As the BI Revive System Parameters (brought by BI itself) are executing functions on missionstart it has to be tested if you can alter these settings after missionstart with another function call or similar.
This would be an important thing for people who use ACE and need to reliable deactivate the BI Revive System (currently via editing the default value in the mission_params.hpp).

**Comment by veteran29** [comment]
I was able to get this working with these changes: https://github.com/veteran29/kp_liberation/compare/65b4d60817c5cad81e6dc941e92e85d20c963340...0731dc63449dd1090129685ebe37eaa964fdd004

I'm not sure if `ReviveMode = 0;` in description.ext was necessary but I wanted to be sure that `bis_fnc_reviveInit` is not run automatically on server init as it registers event handlers on `player` and it would be duplicated when called manually in [init.sqf](https://github.com/veteran29/kp_liberation/commit/0731dc63449dd1090129685ebe37eaa964fdd004#diff-7378c8a41799082c2f8348efef185a72R19).

I tested this briefly on local listen server. JiP and dedi were not be tested.

I hope this will be usefull to you. Cheers.
